Resources/Prototypes/_CM14/Entities/Structures/Walls/girder.yml

```
@@ -0,0 +1,14 @@
+- type: entity
+  parent: Girder
```

This should parent CMEntityBase as well so we can categorize them later between cm and non-cm entities.
```suggestion
  parent: [Girder, CMEntityBase]
```